clusterloader2/cmd/clusterloader.go

```diff
@@ -85,6 +85,7 @@ func initClusterFlags() {
 	flags.StringSliceEnvVar(&clusterLoaderConfig.ClusterConfig.MasterIPs, "masterip", "MASTER_IP", nil /*defaultValue*/, "Hostname/IP of the master node, supports multiple values when separated by commas")
 	flags.StringSliceEnvVar(&clusterLoaderConfig.ClusterConfig.MasterInternalIPs, "master-internal-ip", "MASTER_INTERNAL_IP", nil /*defaultValue*/, "Cluster internal/private IP of the master vm, supports multiple values when separated by commas")
 	flags.BoolEnvVar(&clusterLoaderConfig.ClusterConfig.APIServerPprofByClientEnabled, "apiserver-pprof-by-client-enabled", "APISERVER_PPROF_BY_CLIENT_ENABLED", true, "Whether apiserver pprof endpoint can be accessed by Kubernetes client.")
+	flags.IntEnvVar(&clusterLoaderConfig.ClusterConfig.ParallelEventDeletions, framework.ParallelEventDeletionsName, "PARALLEL_EVENT_DELETIONS", 0, "Number of events to delete in parallel before deleting namespace. For faster test cleanup. Zero means just let namespace deletion implicitly clean up events - which is slower for large event counts")
```

Comment:
I'm against this change.

If you want to speed up events deletion, you can already do that. it's a knob that is exposed from kube-apiserver. It's here:
https://github.com/kubernetes/kubernetes/blob/2c1c1b75465e93ecc49d0cd59efdfa1b0bdb3bbc/staging/src/k8s.io/apiserver/pkg/server/options/etcd.go#L127

This does exactly what you want at the lower level.

Reply:
@wojtek-t In the past few days, I've worked with two users who could not use the lower-level feature.  Here at Microsoft, that will be common.  E.g. if customers want to use ClusterLoader2 to benchmark an AKS cluster, they cannot set the kube-apiserver flag, because we don't expose it to them. The same will also be true of other cloud providers who provide management or control of API server config - they might not expose that flag to customers.

Obviously, I _could_ change the default value of the API server flag for _all_ AKS, but I'm reluctant to make and test that change, and apply it to _everyone_, just to benefit the smaller number who may want to benchmark their clusters with CL2.

So what do you think about re-evaluating this PR, on that basis?  If it helps, I can modify the prompt message in this PR, to point out that the kube-apiserver flag is the preferred solution, and users should only use the CL2 flag if they don't have access to the api-server flag.

Reply:
OK - I buy that.

Let's make change the help message for the flag as well as pring some Warning in the logs if the value is set. 

[I didn't look into the code itself - so the above is purely on the conceptual level.]

Reply:
On second thoughts, we've noticed that some auditing software doesn't particularly like deletion of events.  (The action gets classified as suspicious behaviour - like to how modification of logs would be similarly classified.).  So on that basis I'm going to abandon this PR.

To others who may read this in the future: if you want faster deletion of namespaces that have lots of stuff in them, your current options are 
(a) manually increase delete-collection-workers as wojtek-t describes above (if you have that level of access to your A
PI server)
(b) persuade your API server administrator to do the change for you, OR
(c) submit a PR for the K8s API server itself to increase the default (I'm not sure whether you'll succeed with that - presumably there is a reason why the current default is 1)